etc/linux-systemd/system/syncthing@.service

```diff
@@ -9,6 +9,7 @@ ExecStart=/usr/bin/syncthing -no-browser -no-restart -logflags=0
 Restart=on-failure
 SuccessExitStatus=3 4
 RestartForceExitStatus=3 4
+RuntimeDirectory=syncthing
```

Comment:
As far as I can see we never use the runtime directory, unless the user manually creates a unix socket. Meaning in 99.9% this is just an empty directory. And even for those using the socket, it seems to be common practice to just place a single file directly into `run` according to what I see there on my system.